contracts/BinaryOptionMarket.sol

```diff
@@ -404,72 +229,38 @@ contract BinaryOptionMarket is Owned, MixinResolver, IBinaryOptionMarket {
         require(!_manager().paused(), "This action cannot be performed while the contract is paused");
     }
 
-    function requireActiveAndUnpaused() external view {
-        _systemStatus().requireSystemActive();
+    function requireUnpaused() external view {
         _requireManagerNotPaused();
     }
 
     /* ========== MUTATIVE FUNCTIONS ========== */
 
-    /* ---------- Bidding and Refunding ---------- */
+    /* ---------- Minting ---------- */
 
-    function _updatePrices(
-        uint longBids,
-        uint shortBids,
-        uint _deposited
-    ) internal {
-        (uint256 longPrice, uint256 shortPrice) = _computePrices(longBids, shortBids, _deposited);
-        prices = Prices(longPrice, shortPrice);
-        emit PricesUpdated(longPrice, shortPrice);
-    }
-
-    function bid(Side side, uint value) external duringBidding {
+    function mint(uint value) external duringMinting {
         if (value == 0) {
             return;
         }
 
-        _option(side).bid(msg.sender, value);
-        emit Bid(side, msg.sender, value);
+        uint valueAfterFees = value.multiplyDecimalRound(_feeMultiplier);
 
-        uint _deposited = _incrementDeposited(value);
-        _sUSD().transferFrom(msg.sender, address(this), value);
+        _mint(msg.sender, valueAfterFees);
 
-        (uint longTotalBids, uint shortTotalBids) = _totalBids();
-        _updatePrices(longTotalBids, shortTotalBids, _deposited);
+        _incrementDeposited(value);
+        _manager().transferSusdTo(msg.sender, address(this), value);
```

Comment:
all bidding and price logic is obsolete. The mint method takes the sUSD amount and mints 1 long and 1 short option minus the 1% fee for every sUSD.